libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1313,3 +1316,19 @@ void RC_Channels::convert_options(const RC_Channel::aux_func_t old_option, const
         }
     }
 }
+
+bool RC_Channel::radio_input_changed(uint16_t rc_in)
+{
+    if (rc().has_had_rc_receiver() && !rc().ignore_receiver()) {
```

Comment:
Going against my previous advice, I'm not sure if we should have the, "rc().has_had_rc_receiver()" part.

Reply:
OK
radio_input_changed function is called when only has_had_rc_receiver() is true because it is checked at the beginning of read_input.
so I removed it again.